tests/test_search.py

```diff
@@ -165,10 +165,15 @@ def test_doc_by_field(self):
         self.assertEqual(self.searcher.doc('CACM-3134').docid(),
                          self.searcher.doc_by_field('id', 'CACM-3134').docid())
 
+    def test_phrase_df(self):
+        self.assertEqual(74, self.searcher.phrase_df("inform retriev"))
```

Comment:
I assume you've sanity checked the output by other means? i.e., that df _is_ correct?